src/config.rs

```diff
@@ -252,6 +255,7 @@ pub fn construct_config(pc: ProxyConfig) -> Result<Config, Error> {
 
         enable_original_source: parse(ENABLE_ORIG_SRC)?,
         proxy_args: parse_args(),
+        disable_impersonated_identity: parse_default(DISABLE_IMPERSONATED_IDENTITY, false)?,
```

Comment:
```suggestion
        disable_impersonated_identity: parse_default(DISABLE_IMPERSONATED_IDENTITY, false)?,
```

Can this just be `impersonate_identity`

Reply:
OK, will change to ENABLE_IMPERSONATE_IDENTITY, and default to true.